src/api/authorData.js

```diff
@@ -38,9 +38,9 @@ const deleteSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
 const updateAuthor = () => {};
 
 // TODO: GET A SINGLE AUTHOR'S BOOKS   use thissss!!!!!!!!!!!
-const getbooksByAuthor = (authorId) => new Promise((resolve, reject) => {
-  axios.get(`${dbUrl}/books.json?orderBy="author_id"&equalTo="${authorId}"`)
-    .then((response) => resolve(Object.values(response.data)))
+const getbooksByAuthor = (firebaseKey) => new Promise((resolve, reject) => {
```

Comment:
Why did we change the name of this parameter? It's good to describe our variables as best we can to make well-documented code!